src/online2bin/online2-tcp-nnet3-decode-faster_word_timestamps.cc

```diff
@@ -404,6 +422,13 @@ int main(int argc, char *argv[]) {
             CompactLattice lat;
             decoder.GetLattice(true, &lat);
             std::string msg = LatticeToString(lat, *word_syms);
+
+            auto transcript_time = high_resolution_clock::now();
+            auto duration = duration_cast<milliseconds>( transcript_time - transcription_start ).count();
+            std::string current_duration = std::to_string(duration);
+
+            global_message = "{\"blockend\": true, \"time_from_beginning\": " + current_duration + ", " + global_message;
```

Comment:
What is the difference between `current_duration` and `current_timestamp`? Can we give it more explanatory naming? Maybe `block_duration` instead of `current_duration`? I'm not sure if I get the meaning of the variable correctly. Also put underscore in `block_end` so we're consistent.

Reply:
current duration is time elapse since the beginning of Kaldi decoding, current_timestamp serves a purpose to be a unique identifier for each block, I've change the naming from current_duration to block_identifier as it is indeed more explanatory that way. I've also changed the blockend to block_end.